React/wbooks/src/app/config/api.js

```diff
@@ -1,7 +1,9 @@
 import axios from 'axios';
+import * as localstorageService from '../../services/localstorageService';
 
 const api = axios.create({
-  baseURL: 'https://wbooks-api-stage.herokuapp.com/api/v1'
+  baseURL: 'https://wbooks-api-stage.herokuapp.com/api/v1',
+  headers: { 'Authorization': localstorageService.retrieveUserFromLocalStorage() }
```

Comment:
Are u setting all user object to the header?? that's not right! just the token is needed!